### ISSUE TITLE
Getting a loading error on IE 11

### ISSUE THREAD

**Issue by MartijnR** [enketo/enketo-legacy#341]
Reported by Okal in https://github.com/kobotoolbox/enketo-express/issues/352

To reproduce

Open a webform in IE.
Fill it and save as a draft.
Open the draft from the sidebar.
Message

Loading Error

Error occured during the loading of this form. It is recommended not to edit this record until this is resolved.

Please contact supoprt@enketo.org with the link to this page and the error message below:
 Error trying to parse XML record. No such interface supported.
Environment

IE 11 on Windows 7 with Service Pack 1.
Webform URL: https://uibgw.enketo.org/webform

**Comment by MartijnR** [comment]
Thanks! 

If this is specific to IE11, this will not be fixed (and otherwise probably also not, because this app is being replaced by Enketo Express).

**Comment by okal** [comment]
Fair enough. We're transitioning to Express, but haven't completed the move yet. I grepped and found this in Enketo Core, so I thought it might arise even on Express. Are you aware of any workarounds we can share with our users, other than "Use a different browser"?

**Comment by MartijnR** [comment]
What issue in Enketo Core did you find?

If it's specific to IE11 (which is also no longer supported in Enketo Express btw), then 'using a modern browser', such as Microsoft Edge is the only solution, I'm afraid. 

Sorry about that. It's a few months too early to loose IE11 support. It must be an old issue that for some reason was never reported before (when this repo was still active).

However, let me check if the issue occurs in Microsoft Edge. If so then, it will be fixed in Enketo Express (not Enketo Legacy - because it would take many days to upgrade enketo-core in Enketo Legacy due to some huge changes in Enketo Core)

**Comment by MartijnR** [comment]
Does not occur in Microsoft Edge (and I confirmed the issue in IE11)
